_includes/footer.html

```diff
@@ -1,32 +1,21 @@
-<footer class="footer grid h-card">
-  <data class="u-url" href="{{ "/" | relative_url }}"></data>
+<footer class="footer grid">
+  <data href="{{ "/" | relative_url }}"></data>
```

Comment:
Lol what.

Reply:
(I was "lol what"-ting about the `{{ '/' | relative_url }}` construct, but actually, this should be an `<a>` instead, right?)

Reply:
Thanks for calling this out! This comes from the [minima theme](https://github.com/jekyll/minima/commit/00e75a523bf3be753d8b3ea438437a9d2d7fc2cb#diff-7693eb643e8fe8b8d46b407909b408ceR2) I copied over and honestly I don't know what its doing. I know minima uses [jekyll-seo-tag](https://github.com/jekyll/jekyll-seo-tag) (which we are still using) to do some seo and things with blog authors like adding meta tags and microformats—semantic web type stuff but I never bothered to check on this. Google got me [h-card](http://microformats.org/wiki/h-card), so looks like I broke it even by leaving it in.

We probably should take it out since I don't think we need it and I don't really want to learn how it works. I'll add a commit to remove it.

Yes, also the `relative_url` is funny. I don't know why they use that, I've also seen `{{ site.baseurl}}/` and `{{ '/' | prepend: site.baseurl }}`. idk. But to your point, I'll throw the ODCa footer branding in an `<a>` tag too.